.github/codecov-ut-config/codecov.yml

```diff
@@ -0,0 +1,25 @@
+codecov:
+  require_ci_to_pass: yes
```

Comment:
i think we don't need ci to pass to show this coverage

Reply:
modified